node/src/chain_spec.rs

```diff
@@ -329,6 +377,52 @@ fn testnet_genesis(
                 // TODO initialize accounts with Mangata token
                 .flat_map(|_| vec![])
                 .collect(),
+            created_tokens_for_staking: {
+                let mut created_tokens_for_staking_token_1: Vec<(AccountId, u32, u128)> =
+                    staking_accounts
+                        .iter()
+                        .cloned()
+                        .map(|x| {
+                            let (who, _, _, token_id, initial_amount, _, _) = x;
```

Comment:
consider moving values extraction into map statement

Reply:
Created trello task for it.